AutomatticTracks/build.gradle

```diff
@@ -22,7 +24,7 @@ dependencies {
 
 android {
     compileSdkVersion 26
-    buildToolsVersion "26.0.2"
+    buildToolsVersion "28.0.3"
```

Comment:
@malinajirka 

> The `publishToMavenLocal` task doesn't work anymore.

Uh, yeah. Sorry about that. I fixed it in the other PR #44. I wanted to focus on the Gradle upgrade in this PR, in case I screwed things up. 

I'll cherry pick that to this PR. 

> According to the gradle plugin documentation it's no longer needed to explicitly specify android.buildToolsVersion. So I think we can just simply remove it. Wdyt?

Sure! I'll remove it.